binding.gyp

```diff
@@ -15,8 +18,55 @@
             },
             'libraries' : ['-lz -lrdkafka -lrdkafka++']
           }
+        ],
+        [
+          'OS=="linux"',
+          {
+            'cflags': ['-Wall', '-Wno-sign-compare', '-Wno-missing-field-initializers'],
+            'libraries' : ['-lz -lrdkafka -lrdkafka++']
```

Comment:
Yep it does. I've deleted 'rdkafka' libs from my `/usr/local` and the build started failing. So it builds librdkafka, but doesn't link to the built version, it still tries to link to the installed version.